spec/meta_inspector/redirections_spec.rb

```diff
@@ -18,6 +18,25 @@
       end
     end
 
+    context "when there are too many redirects" do
+      before(:all) { WebMock.enable! }
+      after(:all)  { WebMock.disable! }
+
+      before do
+        12.times { |i| register_redirect(i, i+1) }
+      end
+
+      let(:perform_request) do
+        MetaInspector.new("http://example.org/1")
+      end
+
+      it "it raises an error" do
+        expect {
+          perform_request
```

Comment:
As we're not using this `perform_request` in more places, I would remove it and just use `MetaInspector.new("http://example.org/1")` directly, this way we can remove the 3 lines required to its definition above.